app/assets/javascripts/adoption_form.js

```diff
@@ -66,5 +66,30 @@ $( function() {
         select: itemSelected,
         source: remoteSource
       });
+
+    var isRefEditing = false;
+
+    $('#references').on('click', 'a#toggle-edit-ref', function() {
+      if (isRefEditing) {
+        // find all disabled fields in .edit-reference and enable them
+        $('form.edit-reference .to-disable').attr('disabled', 'disabled');
```

Comment:
Not my forte, and is probably readily apparent during use, but is this right: `.attr('disabled', 'disabled');`? It looks a bit suspicious. Two things:
- the comment says it's enabling them but it looks like it's setting it `disabled`,
- in XHTML `disabled` an attribute, but in HTML5 it's a property - should this be a property here? I imagine functionally equivalent, but not sure what the browser support story is. E.g. `.prop( "disabled", true );`

Reply:
@tobymurray 
- The comment is wrong.
- You are right, it should be .prop() according to jQuery1.6+ documentation. 
@markottaviani 
Do you want me to work through Toby's suggestions?